doc/user-manual/tools/emacs-mode.rst

```diff
@@ -181,6 +181,9 @@ Global commands
 :kbd:`C-c C-x C-s`
      Switch to a different Agda version
 
+:kbd:`C-c C-x C-x`
+     When in a literate Agda file, switch to the associated mode, e.g. LaTeX.
```

Comment:
Naturally, this is a one-way trip, there is no key combination to get back to the Agda mode?

Reply:
In general yes, but there is a feature to bind the return: The last two items in `("lagda" latex-mode latex-mode-map "\C-c\C-x\C-x")` in `agda2-literate-modes`. This is a hack, since now that will always take you to Agda mode even if it's not a literate Agda file, but it's probably pretty unlikely that do it by accident.